src/python/build_management/revisions.py

```diff
@@ -694,7 +697,7 @@ def write_revision_to_revision_file(revision_file, revision):
   """Writes a revision to the revision file."""
   try:
     with open(revision_file, 'wb') as file_handle:
-      file_handle.write(bytes(revision))
+      file_handle.write(str(revision))
```

Comment:
This is potentially a slight behavior change, but I don't expect it to cause any issues.